core/iwasm/fast-jit/cg/x86-64/jit_codegen_x86_64.cpp

```diff
@@ -186,41 +186,45 @@ jit_codegen_interp_jitted_glue(void *exec_env, JitInterpSwitchInfo *info,
         }                                                       \
     } while (0)
 
-#define ASSERT_I32_REG_NO(no) \
-    bh_assert(no >= 0 && (uint32)no < sizeof(regs_i32) / sizeof(regs_i32[0]))
+#define CHECK_I32_REG_NO(no)                                                   \
+    do {                                                                       \
+        if (!(no >= 0 && (uint32)no < sizeof(regs_i32) / sizeof(regs_i32[0]))) \
```

Comment:
```suggestion
        if ((uint32)no >= sizeof(regs_i32) / sizeof(regs_i32[0]))) \
```
just realized, may be we can simplify the condition.

Reply:
OK, sounds better.